--- jless-0.9.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ jless-0.9.0/Cargo.toml	2023-08-03T16:56:44.805791+00:00
@@ -31,11 +31,13 @@
 version = "4.0"
 features = ["derive"]
 
-[dependencies.clipboard]
-version = "0.5"
+[dependencies.clipboard]
+version = "0.8"
+package = "copypasta"
 
-[dependencies.isatty]
-version = "0.1"
+[dependencies.isatty]
+version = "0.2"
+package = "atty"
 
 [dependencies.lazy_static]
 version = "1.4.0"
